conf/messages.en

```diff
@@ -882,24 +866,25 @@ title.turkey=Turkey
 title.turkmenistan=Turkmenistan
 title.turks_and_caicos_islands=Turks and Caicos Islands
 title.tuvalu=Tuvalu
+title.ussr=USSR
 title.uganda=Uganda
 title.ukraine=Ukraine
 title.united_arab_emirates=United Arab Emirates (Abu Dhabi, Ajman, Dubai, Fujaairah, Ras al Khaimah, Sharjah and Umm al Qaiwain)
 title.umm_al_qaiwain=Umm al Qaiwain
-title.united_kingdom_of_great_britain_and_northern_ireland_the=United Kingdom (Great Britain and Northern Ireland)
-title.united_states_minor_outlying_islands=United States minor outlying islands (including Baker Island, Howland Island, Jarvis Island, Johnson Atoll, Kingman Reef, Midway Islands, Navassa Island, Palmyra Atoll and Wake Island)
-title.united_states_of_america=United States of America (including Puerto Rico)
+title.united_kingdom_of_great_britain_and_northern_ireland_the=United Kingdom
+title.united_states_of_america=United States of America
```

Comment:
this is "United States" in the spreadsheet

Reply:
👍